bridge/client/app/_components/ktb-sequence-tasks-list/ktb-sequence-tasks-list.component.ts

```diff
@@ -30,52 +25,32 @@ export class KtbSequenceTasksListComponent implements OnInit, OnDestroy {
     if (this._tasks !== value) {
       this._tasks = value;
       this.focusLastSequence();
-    }
-  }
-
-  @Input()
-  get stage(): string | undefined {
-    return this._stage;
-  }
-
-  set stage(value: string | undefined) {
-    if (this._stage !== value) {
-      this._stage = value;
-      this.focusLastSequence();
+      this._changeDetectorRef.markForCheck();
     }
   }
 
   @Input()
   get focusedEventId(): string | undefined {
     return this._focusedEventId;
   }
-
   set focusedEventId(value: string | undefined) {
     if (this._focusedEventId !== value) {
       this._focusedEventId = value;
     }
+    if (!value) {
+      this.focusLastSequence();
+    }
+    this._changeDetectorRef.markForCheck();
```

Comment:
Is this necessary? If the input changes (reference equality), then the change detection should trigger anyway.

Reply:
In this component you are right, it is not needed. But it's weird because in another component (`ktb-heatmap-tooltip.component.ts`) this was necessary.

Reply:
Kudos for removing `ChangeDetectorRef` :+1: